stories/topics/proc-aap-aws-uninstall.adoc

```diff
@@ -26,12 +27,13 @@ If you did not install extension nodes, skip theses steps.
 .. Confirm the deletion.
 . Extension nodes have been removed
 
-== Uninstall {AAPonAWS}
+== Uninstalling {AAPonAWS}
 
 .Procedure
 . In the AWS UI, navigate to the link:https://us-east-1.console.aws.amazon.com/cloudformation/home?region=us-east-1[cloudformation page].
```

Comment:
Should this URL be a bit more generic - ie not specify a region?

Reply:
I think this is the region it's all set up on.  I've seen it mentioned lots of times in the AWS Slack channel.